src/web/command_control/client/components/CommandControl.tsx

```diff
@@ -463,6 +465,16 @@ export default class CommandControl extends React.Component {
 
 		this.setupMapLayersPanel()
 
+		const emptyMission = {
+			id: '',
+			name: '',
+			runs: {},
+			runIdIncrement: 1,
+			botsAssignedToRuns: {},
+			runIdInEditMode: ''
+		}
+		this.updateMissionHistory(emptyMission)
```

Comment:
Why do we need an empty mission?

Reply:
I think it is fine where it is now. We know that we have some optimization that we need to do for it anyways.